### PR TITLE
chore!: Remove pre-built browser bundle & browser tests

### LINKED ISSUE #8: Remove browser code
We removed it from theming-log and I think we should remove it from here.

### PR DESCRIPTION
**Pull request by sttk**
This PR removes code for Web browsers.

Closes #8

### CODE REVIEW

**Comment by sttk** [comment]
@phated I forgot to replace `chai` to `expect` for test code, and I've done it. Please review this PR.

File: README.md [hunk 1]
```diff
@@ -114,7 +102,7 @@ Executes the _fn_ function for all properties.
 
 ## License
 
-Copyright (C) 2016-2021 Gulp Team.
+Copyright (C) 2016-2022 Gulp Team.
 
 This program is free software under [MIT][mit-url] License.
 See the file LICENSE in this distribution for more details.
```
Comment (phated) [inline README.md:108]:
```suggestion
MIT
```

Comment (phated) [inline README.md:108]:
```suggestion
MIT
```

**Review by phated — APPROVED** [review]
I made one change to the README but this looks great! Thanks!

**Pull request merged**